src/lib/canvas/canvas.js

```diff
@@ -19,20 +20,33 @@ const Container = styled.div`
 class Canvas extends React.Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      creatorListId: props.creatorListId,
```

Comment:
why do you save props to state? I don't see that you change the state of creatorListId and creators in this component.
Moreover, often it's a bad practice to mess props with state.

Reply:
you are right